### PR TITLE
[streamango] fix embed links by making title optional

### PR DESCRIPTION
**Pull request by steebchen**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

Use empty title as default to fix embed links where no og:title is set. Tried fatal=False and default=None but these two throwed another error.

### CODE REVIEW

**Comment by dstftw** [comment]
1. Add a test.
2. Use video_id.

**Comment by steebchen** [comment]
1. A test with only `info_dict.title: ''` or do I need to add an md5 and some other stuff?
2. Use video_id where exactly?

**Comment by dstftw** [comment]
1. A test should test what you are trying to fix.
2. Where you use `''`, where else?

**Comment by steebchen** [comment]
1. I know that. I was asking whether I can only use info_dict with title and without md5 and other stuff.
2. Ah, of course. Brain had stopped working.

File: youtube_dl/extractor/streamango.py [hunk 1]
```diff
@@ -22,6 +22,13 @@ class StreamangoIE(InfoExtractor):
             'title': '20170315_150006.mp4',
         }
     }, {
+        'url': 'https://streamango.com/embed/ptcbfodbpadmnarm/Koty_i_zaloty_0001',
```
Comment (dstftw) [inline youtube_dl/extractor/streamango.py]: This video has `og:title`.

Comment (steebchen) [inline youtube_dl/extractor/streamango.py]: Sorry. Fixed

**Pull request closed without merging**